core/src/Cabana_AoSoA.hpp

```diff
@@ -112,6 +112,10 @@ class AoSoA
     // AoSoA type.
     using aosoa_type = AoSoA<DataTypes, DeviceType, VectorLength, MemoryTraits>;
 
+    // Host mirror type.
+    using host_mirror_type =
+        AoSoA<DataTypes, Kokkos::HostSpace, VectorLength, MemoryTraits>;
```

Comment:
Where did you mean to use that public member type?

Reply:
I was going to add it in for use with the mirror return types but it turns out we don't need to use it (unless we want to add a host specific specialization).

Where I actually think it's useful is so I can store a class pointer to a mirror, without manually constructing the type (in user code).

In VPIC I now do:

```
class foo {
  // set this later using Cabana::create_mirror_view
  particle_t::host_mirror_type particles_host;
};
```

Reply:
Not sure it does what you want for unmanaged memory though.

Reply:
Hmm. I guess I'm not sure either..

Reply:
I agree with it not working well under unmanaged memory -- good catch, thanks. 

I had a chat with Stuart and we decided it might be better to avoid inheriting any traits at all (which I just pushed) -- what are your thoughts on how this should look?

We can also just not merge this, and I can do some declytyping / hard coding in user space

Reply:
For reference, not inheriting traits would then provide the exact same behavior as `Kokkos::View<>::HostMirror` hence why we chose to remove them.